lib/data/data_source/remote/authentication_remote_source_impl.dart

```diff
@@ -73,11 +85,11 @@ class AuthenticationRemoteSourceImpl implements AuthenticationRemoteSource {
   }
 
   @override
-  Future<Success> updateUser(UpdateUserDto dto) async {
+  Future<Success> updateUser(UserProfileDto dto) async {
     try {
       final user = FirebaseAuth.instance.currentUser;
       if (user != null) {
-        await user.updateDisplayName(dto.username);
+        await user.updateDisplayName(dto.name);
```

Comment:
Now that we have a document for the user profile, we need to fetch and update the document rather than the Firebase authentication parameters.